docs/docs/rad-lab-ui/ui_configure/ui-meta.md

```diff
@@ -61,10 +67,56 @@ variable "boot_disk_size_gb" {
 ```
 
 ## Updatesafe
+
 After a module has been deployed (the project created and resources created within it), some Terraform updates can be **destructive**. To warn the users that they may be performing a potentially destructive action, the `updatesafe` key word is added (or omitted) to the `UIMeta`.
 
 Any time a user makes an update to an existing deployment, the `updatesafe` values are checked. If **ANY** of the changed variables do **NOT** have the `updatesafe` keyword, the user will be warned that they likely will lose data by applying the update. Put another way, only if **ALL** the changed variables are `updatesafe` will the prompt be skipped.
 
+## Mandatory
```

Comment:
Can we rename the key word from `mandatory` to `required`? It's a much more common term for this.